### PR TITLE
Add explanations of kinesis_firehose and cloudwatch_logs

### PR DESCRIPTION
**Pull request by SpringMT**
Added `kinesis_firehose` and `cloudwatch_logs` plugins into Core Fluent Bit Plugin.

* https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit#new-higher-performance-core-fluent-bit-plugin
* https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit#new-higher-performance-core-fluent-bit-plugin

So, add descriptions about them.

### CODE REVIEW

**Review by agup006 — APPROVED** [review]

**Comment by agup006** [comment]
+1 thanks, I think we should look at updating that page with the latest information as well. giving @PettitWesley before I merge

File: installation/aws-container.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ Currently, the [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) i
 * [Amazon Kinesis Firehose](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit)
 * [Amazon Kinesis Streams](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit)
 
+Fluent Bit includes CloudWatch Logs plugin named `cloudwatch_logs` and Kinesis Firehose plugin named `kinesis_firehose` which are higher performance than Go plugins.
+
+* [Amazon CloudWatch](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch)
+* [Amazon Kinesis Data Firehose](https://docs.fluentbit.io/manual/pipeline/outputs/firehose)
```
Comment (PettitWesley) [inline installation/aws-container.md:16]: @SpringMT we also now have a core kinesis_streams plugin too: https://docs.fluentbit.io/manual/pipeline/outputs/kinesis

Please add that one and then we can merge this.

**Review by PettitWesley — CHANGES_REQUESTED** [review]

File: installation/aws-container.md (hunk 1 above)
Comment (SpringMT) [reply installation/aws-container.md:16]: Added `kinesis_streams` explanation.
Thx!

**Review by PettitWesley — CHANGES_REQUESTED** [review]
@SpringMT Awesome, sorry, one last thing is your commit messages. Notice the pattern for all existing commits: https://github.com/fluent/fluent-bit-docs/commits/master

So for this case, something like:

```
aws: add links to AWS plugins for the aws distro
```

**Comment by PettitWesley** [comment]
@SpringMT actually, I just realized- the core S3 plugin should also be in this list right?

**Comment by SpringMT** [comment]
I agree adding S3 plugin.

How about this?

```
## Plugins
Currently, the [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) image contains Go Plugins for:

* [Amazon CloudWatch Logs](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit)
* [Amazon Kinesis Firehose](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit)
* [Amazon Kinesis Streams](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit)

Fluent Bit includes Amazon CloudWatch Logs plugin named `cloudwatch_logs`, Amazon Kinesis Firehose plugin named `kinesis_firehose` and Amazon Kinesis Data Streams plugin named `kinesis_streams` which are higher performance than Go plugins.

* [Amazon CloudWatch](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch)
* [Amazon Kinesis Data Firehose](https://docs.fluentbit.io/manual/pipeline/outputs/firehose)
* [Amazon Kinesis Data Streams](https://docs.fluentbit.io/manual/pipeline/outputs/kinesis)

Also, Fluent Bit includes S3 output plugin named `s3`.
* [Amazon S3](https://docs.fluentbit.io/manual/pipeline/outputs/s3)
```

**Comment by PettitWesley** [comment]
@SpringMT I like it

**Comment by SpringMT** [comment]
I fixed commit messages and add S3 plugin explanation.

**Comment by agup006** [comment]
@PettitWesley if you change to approval I can merge

**Review by PettitWesley — APPROVED** [review]

**Pull request merged**